style.css

```diff
@@ -0,0 +1,830 @@
+  align-items: center;
+}
+
+.index-nav {
+  display: flex;
+  flex: 0 0 100%;
+  justify-content: flex-start;
+  flex-flow: column wrap;
+  color: #f42;
+}
+
+.second-row div {
+  margin-left: 2%;
+  margin-top: 1%;
+}
+
+.hidden-on-mobile-and-desktop {
+  position: absolute;
+  clip: rect(1px 1px 1px 1px);
+  clip: rect(1px, 1px, 1px, 1px);
+}
+
+/* Tablet Screen Sizes */
+@media only screen and (min-width: 480px) {
+  .black-nav {
+    display: flex;
+    align-items: center;
+    flex: 0 0 100%;
+    background-color: #131313;
+    color: #555;
+    height: 30px;
+    font-size: 12px;
+    justify-content: space-between;
+  }
+
+  .white-nav {
+    display: flex;
+    align-items: center;
+    background-color: white;
+    color: #555;
+    height: 30px;
+    font-size: 12px;
+    justify-content: space-between;
+  }
+
+  .white-nav div {
+    justify-self: space-around;
+  }
+
+  .hidden-on-tablet-and-desktop {
+    position: absolute;
+    clip: rect(1px 1px 1px 1px);
+    clip: rect(1px, 1px, 1px, 1px);
+  }
+
+  .hidden-on-mobile-and-desktop {
+    position: unset;
+  }
+
+  .col-m2 {
+    display: flex;
+    flex: 0 0 46%;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .col-m3 {
+    display: flex;
+    flex: 0 0 29%;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+  }
+
+  a[class*="image-"] {
+    flex: 0 0 100%;
+    height: 150px;
+  }
+
+  .article-head {
+    font-size: 100%;
+  }
+
+  .first-row {
+    display: flex;
+    flex: 0 0 60%;
+    align-items: stretch;
+    flex-flow: row wrap;
+  }
+
+  .img-1 {
+    height: 368px;
+  }
+
+  .second-row {
+    display: flex;
+    flex: 0 0 25%;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: space-between;
+  }
+
+  .second-row div {
+    flex: 0 0 100%;
+    margin-left: 2%;
+    display: flex;
+    align-items: stretch;
+  }
+
+  .tablet-nav {
+    display: flex;
+    flex-flow: row nowrap;
+  }
+
+  .tablet-nav-logo {
+    display: flex;
+    background-image: linear-gradient(red, yellow);
+    flex: 0 0 15%;
+  }
+
+  .tablet-nav-right {
+    flex: 100%;
+  }
+}
+
+/* Desktop screen Sizes */
+@media only screen and (min-width: 768px) {
+  a[class*="apple-image"] {
+    flex: 0 0 100%;
+    height: 200px;
+  }
+
+  .article-head {
+    flex: 0 0 100%;
+  }
+
+  .col-l3 {
+    display: flex;
+    flex: 0 0 29%;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .col-l4 {
+    display: flex;
+    flex: 0 0 20%;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .hidden-on-mobile-and-desktop {
+    position: absolute;
+    clip: rect(1px 1px 1px 1px);
+    clip: rect(1px, 1px, 1px, 1px);
+  }
+
+  .white-nav-1 {
+    display: flex;
+    align-items: center;
+    background-color: white;
+    color: #555;
+    height: 50px;
+    font-size: 20px;
+    justify-content: space-between;
+  }
+}
```

Comment:
That css file doesn't take in consideration all the home page breakpoints..
Please, be sure to identically represent the original page.